Frontend/ui-driver/src/Resource/Styles/Color.purs

```diff
@@ -422,9 +416,6 @@ greenGrey = "#268C6E"
 transparentGrey :: Color
 transparentGrey = "#66888888"
 
-transparent :: Color
```

Comment:
Removed ones are not being used? It's okay to keep color in color files. Now or later we might get the same color

Reply:
I refactored the colors files. moved colors to common which are used in both user and driver also imported common colors in both customer and driver.